inference-engine/src/mkldnn_plugin/nodes/psroi.cpp

```diff
@@ -262,13 +276,38 @@ class PSROIPoolingImpl: public ExtLayerBase {
         return OK;
     }
 
-    inline float bilinear_interp(const float* data, const float x, const float y, const int width) {
+    StatusCode execute(std::vector<Blob::Ptr>& inputs, std::vector<Blob::Ptr>& outputs,
+                       ResponseDesc *resp) noexcept override {
+        float* dst_data = outputs[0]->buffer();
+        const float *bottom_data_fp32 = inputs[0]->buffer();
+        const float *bottom_rois_beginning = inputs[1]->buffer();
+
```

Comment:
Are you sure that these pointers: `dst_data, bottom_data_fp32 , bottom_rois_beginning` are used here?

Reply:
done